packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -594,6 +643,33 @@ export default mixins(
 			},
 		},
 		methods: {
+			credentialSelected (updateInformation: INodeUpdatePropertiesInformation) {
+				// Update the values on the node
+				this.$store.commit('updateNodeProperties', updateInformation);
+
+				const node = this.$store.getters.getNodeByName(updateInformation.name);
+
+				// Update the issues
+				this.updateNodeCredentialIssues(node);
+
+				this.$externalHooks().run('nodeSettings.credentialSelected', { updateInformation });
+			},
+			async prepareScopesNotice(credentialTypeName: string) {
```

Comment:
I don't think we need this anymore since we are not making api requests. can simply be getters for scopes and display name.. also can be moved into nodecredentialstype component.